src/cow-react/modules/limitOrders/state/limitOrdersAtom.ts

```diff
@@ -12,6 +13,7 @@ export interface LimitOrdersState {
   readonly outputCurrencyAmount: string | null
   readonly recipient: string | null
   readonly deadline: number | null
+  readonly orderKind: OrderKind | null
```

Comment:
If you are setting the default on line `28` below, do you still need to have the type to accept `null`?

Reply:
The null was actually because of `useLimitOrdersStateFromUrl` but I guess we can pass default SELL type there also.